lib/usd/ui/layerEditor/stringResources.h

```diff
@@ -106,6 +106,10 @@ const auto kSaveAnonymousIdenticalFiles { create("kSaveAnonymousIdenticalFiles",
 const auto kSaveLayerUsdFileFormatAnn    { create("kSaveLayerUsdFileFormatAnn", "Select whether the .usd file is written out in binary or ASCII. You can save a file in .usdc (binary) or .usda (ASCII) format. Manually entering a file name with an extension overrides the selection in this drop-down menu.") };
 const auto kSaveLayerUsdFileFormatSbm    { create("kSaveLayerUsdFileFormatSbm", "Select whether the .usd file is written out in binary or ASCII") };
 
+const auto kBatchSaveAllRelative { create("kBatchSaveAllRelative", "Convert All to Relative Paths.")};
+const auto kBatchSaveRelativeToScene { create("kBatchSaveRelativeToScene", "Relative to Scene File.")};
+const auto kBatchSaveRelativeToParent { create("kBatchSaveRelativeToParent", "Relative to Parent Layer.")};
+const auto kBatchSaveRelativeTooltip { create("kBatchSaveRelativeTooltip", "This path will be relative to ")};
```

Comment:
You cannot assume that in another language the path can just be appended to the end of this string. That may not be the case. So instead you need to use a marker in the string (see examples in this file using "^1s") and use format to compose the final string.